tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -8,27 +8,75 @@ Feature: full text search
     Given user "Alice" has been created with default attributes and without skeleton files
 
 
-  Scenario Outline: search files using a tag
+  Scenario Outline: search files by tag
     Given using <dav-path-version> DAV path
-    And user "Alice" has uploaded file with content "hello world" to "file1.txt"
-    And user "Alice" has uploaded file with content "Namaste nepal" to "file2.txt"
-    And user "Alice" has uploaded file with content "hello nepal" to "file3.txt"
-    And user "Alice" has created the following tags for file "file1.txt" of the space "Personal":
+    And user "Alice" has created folder "folderWithFile"
+    And user "Alice" has created folder "folderWithFile/subFolder/"
+    And user "Alice" has uploaded file with content "hello world" to "test.txt"
+    And user "Alice" has uploaded file with content "Namaste nepal" to "folderWithFile/fileInsideFolder.txt"
+    And user "Alice" has uploaded file with content "hello nepal" to "folderWithFile/subFolder/fileInsideSubFolder.txt"
+    And user "Alice" has created the following tags for file "test.txt" of the space "Personal":
       | tag1 |
-    And user "Alice" has created the following tags for file "file2.txt" of the space "Personal":
+    And user "Alice" has created the following tags for file "folderWithFile/fileInsideFolder.txt" of the space "Personal":
+      | tag1 |
+    And user "Alice" has created the following tags for file "folderWithFile/subFolder/fileInsideSubFolder.txt" of the space "Personal":
       | tag1 |
     When user "Alice" searches for "Tags:tag1" using the WebDAV API
     Then the HTTP status code should be "207"
     And the search result of user "Alice" should contain only these files:
-      | file1.txt |
-      | file2.txt |
+      | test.txt                |
+      | fileInsideFolder.txt    |
+      | fileInsideSubFolder.txt |
     Examples:
       | dav-path-version |
       | old              |
       | new              |
       | spaces           |
 
 
+  Scenario Outline: search files by tag using new DAV path
```

Comment:
using new dav but i see old as well?

Reply:
Oh, I forget to change the description of the scenario.
Initially, the old Davpath was not working but after some changes in code, it works.